app/controllers/oidc_controller.rb

```diff
@@ -3,17 +3,28 @@
 class OidcController < ApplicationController
   def callback
     reset_session
-    session[:auto_generated_data] = {
-      example_1: 'Lorem ipsum est...',
-      example_2: 'Something or other'
-    }
-    session[:userinfo] = request.env['omniauth.auth'].info
-    redirect_to address_url
-  end
+    sub = request.env['omniauth.auth']['extra']['raw_info']['sub']
```

Comment:
I'm not sure if the oidc controller should deal with the logic around routing/business logic maybe it should pass it on to another controller that handles that logic?